mcs/class/System/mobile_System.dll.sources

```diff
@@ -305,7 +305,6 @@ Mono.Btls/MonoBtlsX509LookupMonoCollection.cs
 Mono.Btls/MonoBtlsX509LookupMono.cs
 Mono.Btls/MonoBtlsX509LookupType.cs
 Mono.Btls/MonoBtlsX509Name.cs
-Mono.Btls/MonoBtlsX509NameList.cs
```

Comment:
My assumption was that this file is there for a reason, that's why Martin should take a look. If it's really dead code then we should of course remove it :)